web/tools.html

```diff
@@ -156,21 +167,21 @@ <h4>Tools</h4>
         </div>
 
         <footer>
-            <!--Buttons from RERUM Home Page. Not properly reflexive currently, and missing the correct font.-->
             <div class="footerButtons">
```

Comment:
use separate html file for the footer(footer.html) as of now the changes are applied only to the tools.html.